src/main/java/saps/scheduler/core/Scheduler.java

```diff
@@ -210,11 +210,11 @@ private void schedule() {
 
     Map<String, List<SapsImage>> tasksByUsers = mapUsers2Tasks(tasks);
 
-    LOGGER.info("Tasks by users: " + tasksByUsers);
+    //LOGGER.info("Tasks by users: " + tasksByUsers);
```

Comment:
@luismafra avoid sending "dead" code to the repository. If these lines are not necessary, remove them.

maybe, instead of removing you can refactor these log lines to indicate the amount of `tasksByUsers`  and `selectedTasks`